ts-test/unit-test.ts

```diff
@@ -41,33 +41,41 @@ type AttachmentType = UnknownType;
 type EventType = UnknownType;
 type MessageType = UnknownType;
 type ReactionType = UnknownType;
+type CommandType = string & {};
```

Comment:
should we abc this ordering too?

Reply:
Unit test is Vish's playground to double check types